File: Python3/Medium/WrongAnswer/destroy-sequential-targets_0.py
class Solution:
    def destroyTargets(self, nums: List[int], space: int) -> int:
        
        nums.sort()
        n = len(nums)
        dp = [1] * n
        for i in range(n):
            for j in range(i):
                if (nums[i] - nums[j]) % space == 0:
                    dp[i] = max(dp[i], dp[j] + 1)
        return max(dp)

File: C++/Medium/WrongAnswer/destroy-sequential-targets_0.cpp
class Solution {
public:
    int destroyTargets(vector<int>& nums, int space) {
        
        int n = nums.size();
        sort(nums.begin(), nums.end());
        int ans = 1;
        for (int i = 0; i < n; ++i) {
            int j = i + 1;
            while (j < n && nums[j] - nums[i] <= space) {
                ++j;
            }
            ans = max(ans, j - i);
        }
        return ans;
    }
};